.github/workflows/broadcast-frontend-hash_modify-file.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+FILE="$1"
+DFX_VERSION="$2"
+NEW_HASH="$3"
+
+awk -v dfx_version="$DFX_VERSION" -v new_hash="$NEW_HASH" '
+BEGIN { OFS="" }
+
+/const WHITELISTED_WASMS: \[&str; [0-9]+\] = \[/ {
```

Comment:
This assumes a lot of things about the Rust code. How about appending the hash to a txt file, and the Rust code uses `include_str!` to load it?

Reply:
https://github.com/dfinity/motoko-playground/pull/185